buildSrc/src/main/kotlin/com/datadog/gradle/config/GradlePropertiesKeys.kt

```diff
@@ -9,4 +9,6 @@ object GradlePropertiesKeys {
 
     // you can set this property from your gradle.properties as: forceEnableLogcat = true | false
     const val FORCE_ENABLE_LOGCAT = "forceEnableLogcat"
+    // you can set this property from your gradle.properties as: buildForCoreTesting = true | false
+    const val BUILD_FOR_CORE_TESTING = "buildForCoreTesting"
```

Comment:
I'm not quite clear how this is going to be used from the RN side, given it either consumes release artifacts or snapshot artifacts (which are release variants as well). Which one of the two is supposed to have this flag enabled?